km/km_filesys.c

```diff
@@ -2284,6 +2306,15 @@ uint64_t km_fs_epoll_ctl(km_vcpu_t* vcpu, int epfd, int op, int fd, struct epoll
    return ret;
 }
 
+uint64_t km_fs_timerfd_create(km_vcpu_t* vcpu, int clockid, int flags)
+{
+   int ret = __syscall_2(SYS_timerfd_create, clockid, flags);
+   if (ret >= 0) {
+      ret = km_add_guest_fd_internal(vcpu, ret, NULL, flags, KM_FILE_HOW_TIMERFD, NULL);
+   }
+   return ret;
+}
+
 static inline int
 km_fs_event_check_errors(km_vcpu_t* vcpu, km_file_t* file, struct epoll_event* events, int maxevents)
```

Comment:
I am not thinking straight where is this getting called from?

Reply:
Doesn't look like it is called from anywhere.

Kind of looks like it was meant to be used as part of snapshot eventfd recovery.